pkg/util/machines_test.go

```diff
@@ -382,6 +382,84 @@ network:
       wakeonlan: true
       dhcp4: true
       dhcp6: true
+`,
+		},
+		{
+			name: "dhcp4+dhcp6+dhcpOverrides",
```

Comment:
add an explicit assertion for when the values are not set, just to verify the default behavior.

Reply:
Updated, thanks!